File: 0633-sum-of-square-numbers/0633-sum-of-square-numbers.cpp
class Solution {
public:
    bool judgeSquareSum(int c) {
        long long int i=0;
        long long int j=sqrt(c);
        while (i<=j){
            long long int ans = i*i + j*j;
            if (ans < c){
                i++;
            }
            else if (ans == c) return true;
            else j--;
        }
        return false;
        
    }
};

File: 0633-sum-of-square-numbers/0633-sum-of-square-numbers.py
class Solution:
    
    
    def judgeSquareSum(self, c: int) -> bool:
        i=0
        j=int(sqrt(c))
        
        while i<=j:
            ans = i*i + j*j
            if ans<c:
                i+=1
            elif ans == c:
                return True
            else: 
                j-=1
                